resources/mysql/fetch_mysql.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+ckmd5() {
+    if [[ $(uname) = Darwin ]]; then
+        test $(md5 -q $1) = $2
+    else
+        test $(md5sum $1 | awk '{ print $1 }') = $2
+    fi
+}
+
+atomic_download() {
+    if [[ ! -e $1 ]]; then
+        curl -o $1.tmp https://cdn.mysql.com//Downloads/MySQL-8.0/$1
+        if $(ckmd5 $1.tmp $2); then
+            mv $1.tmp $1
+        else
+            rm $1.tmp
+        fi
+    fi
+}
+
+mysql_8_0_28_macos11_arm64() {
+    atomic_download mysql-8.0.28-macos11-arm64.tar.gz f1943053b12428e4c0e4ed309a636fd0
+}
+
+mysql_8_0_28_linux_glibc2_17_x86_64_minimal() {
+    atomic_download mysql-8.0.28-linux-glibc2.17-x86_64-minimal.tar.xz 55a7759e25cc527416150c8181ce3f6d
+}
+
+mysql() {
+    mysql_8_0_28_macos11_arm64
```

Comment:
is it possible to detect the arch and only download the one matching the arch?

Reply:
no. we need to embed both https://github.com/bytebase/bytebase/blob/54b2c9c25af1d6c02c4adb021751c3990fed7506/resources/mysql/mysql.go#L20

if we want to just embed one (like pg in release), then we need to add more tags on `go build` to indicate the arch. I don't think it's worthwhile since we just use MySQL for test

Reply:
I think that's fine. We can optimize it later if we see any problem. It's better to be more explicit.

Reply:
Btw, we could use file suffix build tags to decide what to generate and what to download.

https://dave.cheney.net/2013/10/12/how-to-use-conditional-compilation-with-the-go-build-tool

Reply:
Ok, let's move forward for now and address it later.